examples/monitor_dnt.lua

```diff
@@ -0,0 +1,50 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Monitor DNT header status
+
+*Example Heka Configuration*
+
+.. code-block:: ini
+
+    [DNT Usage]
+    type = "SandboxFilter"
+    filename = "examples/monitor_dnt.lua"
+    message_matcher = "Type == 'telemetry'"
+    ticker_interval = 10
+    preserve_data = true
+--]]
+
+require "circular_buffer"
+
+local rows = read_config("rows") or 2880
+local sec_per_row = read_config("sec_per_row") or 60
+local ON  = 1
+local OFF = 2
+local UNK = 3
+
+local c = circular_buffer.new(rows, 3, sec_per_row, true)
+c:set_header(ON, "DNT On")
+c:set_header(OFF, "DNT Off")
+c:set_header(UNK, "DNT Unknown")
```

Comment:
Some of the motivation for this filter is to serve as an example - I think the general pattern of variables for cbuf column numbers + explicitly setting their names makes a good model.